interface.html

```diff
@@ -8,9 +8,18 @@
   </div>
   <main v-bind:class="{ 'select-template': chooseTemplate }">
     <header>
-      <p v-if="chooseTemplate">Template gallery</p>
-      <p v-else>Configure your form</p>
-      <a href="https://help.fliplet.com/article/176" target="_blank">Need help?</a>
+      <p v-if="chooseTemplate">
```

Comment:
I think this can be rewritten to avoid the duplication:

```js
<p>
  <template v-if="chooseTemplate">Template gallery</template>
  <template v-else>Configure your form</template>
  <a>...</a>
</p>
```

Reply:
@squallstar updated